File: app/src/main/java/com/snazhmudinov/movies/models/Trailer.java
package com.snazhmudinov.movies.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

import java.util.List;

/**
 * Created by snazhmudinov on 6/10/17.
 */
@Parcel
public class Trailer {
    @SerializedName("id")
    public Integer id;
    @SerializedName("results")
    public List<Result> results = null;

    public Trailer() {}

    public Integer getId() {
        return id;
    }

    public List<Result> getResults() {
        return results;
    }
}

File: app/src/main/java/com/snazhmudinov/movies/models/Trailer.kt
package com.snazhmudinov.movies.models

import com.google.gson.annotations.SerializedName

import org.parceler.Parcel

/**
 * Created by snazhmudinov on 6/10/17.
 */
@Parcel
class Trailer {
    @SerializedName("id")
    var id: Int? = null
    @SerializedName("results")
    var results: List<Result>? = null
}
